config/flat.py

```diff
@@ -1,5 +1,5 @@
-import os.path
 from lsst.utils import getPackageDir
+import os.path
```

Comment:
Done. I did look at the style guide but did not find anything about system imports going before LSST/PFS ones...